src/__tests__/pages/Home.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../../redux/store';
+import Home from '../../Pages/Home';
+
+test('renders Home component correctly', () => {
+  const { container } = render(<Provider store={store}><Router><Home /></Router></Provider>);
+  expect(container).toMatchSnapshot();
+});
```

Comment:
- [ ] This snapshot test is failing. Please try to make it green ✔️ 
![image](https://github.com/Paulinelali/metric-capstone/assets/46839548/e4c12f18-5fb2-4007-b3c0-cc892c96b235)